lib/Udoit.php

```diff
@@ -142,7 +143,7 @@ public static function scanContent(array $content_items)
 
                 $issue_count++;
                 $state = $quail_issue['state'];
-                //error_log("state in the issue is set as ".serialize($state));
+                $logger->addError("state in the issue is set as ".serialize($state));
```

Comment:
Please remove this line.  It was inadvertently left in from some debugging, and error messages should only be logged when there's a program error, rather than Quail detecting an issue.